model.js

```diff
@@ -0,0 +1,84 @@
+const Model = () => {
+    let data = {}
+    const loadFriends = () =>{
```

Comment:
you do not need to save it in a variable. you are making a module so they should be methods here. just do 

`loadFriends(){`
`$.ajax({...`
`}`